tracing/trace_state.cc

```diff
@@ -208,11 +210,12 @@ void trace_state::build_parameters_map() {
         // queries CQL command, where X is an index of the parameter in a corresponding query and Y is an index of the
         // corresponding query in the BATCH.
         if (prepared_statements.size() == 1) {
-            build_parameters_map_for_one_prepared(prepared_statements[0], vals.query_option_names[0], vals.query_option_values[0], "param");
+            build_parameters_map_for_one_prepared(prepared_statements[0].statement, prepared_statements[0].query_option_names, prepared_statements[0].query_option_values, "param");
```

Comment:
It's a bit of a nitpick but why not to define `auto& stmt_info` like you did for a BATCH below to shorten the `build_parameters_map_for_one_prepared()` call string and avoid typos in the future?

Reply:
no reason - I'll use a helper variable in v2